ui/src/app/components/side-nav.tsx

```diff
@@ -345,7 +337,7 @@ export const SideNav = (props: SideNavProps) => {
         icon='star'
         content='Featured Workspaces'
         onToggleSideNav={() => onToggleSideNav()}
-        href={getLibraryPath()}
+        href={'/library'}
```

Comment:
Not sure if i should rename this to featured-workspaces or keep it as is. As the new class is called featured-workspaces